database.sql

```diff
@@ -0,0 +1,36 @@
+CREATE DATABASE family;
+
+USE family;
+
+CREATE TABLE `users` (
+	`id` INT(10) NOT NULL AUTO_INCREMENT ,
+	`name` VARCHAR(25) NOT NULL,
+	`status` VARCHAR(50) NOT NULL,
+	`age` INT(4) NOT NULL,
+	`address` VARCHAR(100) NOT NULL,
+	`approve_status` TINYINT (1) NOT NULL,
+	`password` VARCHAR(100) NOT NULL,
+	`image` VARCHAR(100),
+	PRIMARY KEY (`id`)
+);
+
+CREATE TABLE `tasks` (
+	`id` INT(10) NOT NULL AUTO_INCREMENT ,
+	`time_created` INT(10) NOT NULL,
```

Comment:
all table columns are related to a date or time must have a TIMESTAMP column type.
You can use mysql DEFAULT and ON UPDATE features to change time. It means when it's created set current timestamp and when it's updated as well 
read this article [https://dev.mysql.com/doc/refman/8.0/en/timestamp-initialization.html](article)